tsstats/templates/index.jinja2

```diff
@@ -49,5 +59,23 @@
   {% endfor %}
   <small>Generated by <a href="https://github.com/Thor77/TeamspeakStats" rel="noopener">TeamspeakStats</a> at {{ creation_time|frmttime }}</small>
 </div>
+
+<script type="text/javascript">
+  document.addEventListener('click', function(event) {
```

Comment:
Thought about for a while now and I don't think we should use event delegation here, as we only need one exact EventListener and are not dynamically modifying the DOM.